docusaurus/docs/user-docs/settings/introduction.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introduction to General Settings
+description: Reviewing the audit logs in Strapi.
+sidebar_position: 2
+---
+
+The _Settings_ feature allows an administrator to adjust all ![Settings icon](/img/assets/icons/settings.svg) _Settings_ from the main navigation of the admin panel, which is divided into 4 sub sections:
+
+- _Global Settings_
+- _Administration panel_
+- _Email plugin_
+- _Users & Permissions plugin_
+
+<ThemedImage
+alt="Custom logo settings"
+sources={{
+    light: '/img/assets/settings/settings_custom-logo.png',
+    dark: '/img/assets/settings/settings_custom-logo_DARK.png',
+  }}
+/>
+
+## Global Settings
+
+The _Global Settings_ in Strapi allows administrators to configure logo in the _Overview & Custom Logo_ interface, manage _API Token_, manage _Media Library_, configure _Internationalization_ locales and configure _Single Sign-On_ and _Webhooks_.
```

Comment:
Perhaps we should make this introduction even more helpful by adding links to those pages, what do you think?